server/controllers/allocationsController.ts

```diff
@@ -288,7 +288,7 @@ export default class AllocationsController {
     crn,
     staffTeamCode,
     staffCode,
-    convictionNumber,
+    convictionNumber: number,
```

Comment:
the convictionNumber was being sent down as a string and causing the stub to not match so had to convert it to a number to fix